src/neptune/handler.py

```diff
@@ -681,12 +681,7 @@ def validate_and_transform_to_extend_format(value):
         so work can be delegated to `extend` method."""
```

Comment:
Can we change `validate_and_transform_to_extend_format` to `transform_to_extend_format`?

Reply:
Done